lib/pliny/error_reporters/rollbar.rb

```diff
@@ -18,9 +18,11 @@ def notify(exception, context:, rack_env:)
       private
 
       def fetch_scope(context:, rack_env:)
-        {
-          request: proc { extract_request_data_from_rack(rack_env) }
-        }
+        scope = {}
+        if rack_env.respond_to?(:body)
```

Comment:
Is this the "accepted" way to detect whether a hash is indeed a `rack_env`? Wouldn't `rack_env.empty?` suffice?